PennMobile/Dining/Views/DiningCell.swift

```diff
@@ -47,19 +47,12 @@ class DiningCell: UITableViewCell {
 // MARK: - Setup Cell
 extension DiningCell {
     fileprivate func setupCell(with venue: DiningVenue) {
-        venueImageView.image = UIImage(named: venue.name.rawValue.folding(options: .diacriticInsensitive, locale: .current))
+        venueImageView.kf.setImage(with: venue.imageURL)
+        titleLabel.text = venue.name
+        updateTimeLabel(with: venue)
         
-        // Use shortened names if on homepage
-        if isHomepage {
-            titleLabel.text = DiningVenueName.getShortVenueName(for: venue.name)
-        } else {
-            titleLabel.text = venue.name.rawValue
-        }
-        
-        updateTimeLabel(with: venue.times)
-        
-        if let times = venue.times {
-            if times.isOpen {
+        if venue.hasMealsToday {
```

Comment:
If the venue has no meals today, shouldn't it read Closed? But then how do we handle instances where we don't know and are awaiting a response from the API?

Reply:
Exactly. We used to say "Closed Today" when there was no data, which was a bit confusing...

Possible fix would be to not filter out the "Closed" meal types?